packages/dds/tree/src/test/simple-tree/api/treeApi.spec.ts

```diff
@@ -1116,15 +1121,55 @@ describe("treeNodeApi", () => {
 			view.initialize({ topLeft, bottomRight, innerPoints: [] });
 
 			const topLeftPoint = view.root.topLeft;
-			const clonedX = TreeBeta.clone<typeof schema.number>(topLeftPoint.x);
+			const clonedX = TreeBeta.clone(topLeftPoint.x);
 			assert.equal(clonedX, topLeftPoint.x, "Number not cloned properly");
 
 			topLeftPoint.metadata = "hydratedPoint";
 			assert(topLeftPoint.metadata !== undefined, "Metadata not set correctly");
-			const clonedMetadata = TreeBeta.clone<typeof schema.string>(topLeftPoint.metadata);
+			const clonedMetadata = TreeBeta.clone(topLeftPoint.metadata);
 			assert.equal(clonedMetadata, topLeftPoint.metadata, "String not cloned properly");
 		});
 
+		// Compile-time test: ensure that the return type of clone is the same as the input type for common cases (nodes, primitives, undefined).
+		function preservesTypesWhenCloning(
```

Comment:
Consider adding a test where a type is explicitly specified to validate that is working as expected.